app/controllers/users_controller.rb

```diff
@@ -121,6 +121,12 @@ def update_passphrase
     end
   end
 
+  def disable_two_step_verification
+    @user.disable_2sv!(current_user)
```

Comment:
Should this be reset_2sv rather than disable? @boffbowsh do you have any thoughts on this?